includes/configs/meta-boxes.php

```diff
@@ -34,20 +34,30 @@
 
 		'fields' => [
 			'event' => [
-				'label'    => esc_html__( 'Event', 'hivepress' ),
-				'type'     => 'select',
-				'options'  => 'emails',
-				'required' => true,
-				'_alias'   => 'post_name',
-				'_order'   => 10,
+				'label'      => esc_html__( 'Event', 'hivepress' ),
+				'type'       => 'select',
+				'options'    => 'emails',
+				'required'   => true,
+				'_alias'     => 'post_name',
+				'_order'     => 10,
+
+				'attributes' => [
+					'data-tokens-render' => hivepress()->router->get_url( 'email_show_tokens' ),
```

Comment:
please re-use existing API - check how attribute meta boxes are refreshed, there's a common JS and PHP that can be re-used by just adding another route